components/logicbutton/metadata/mods.md

```diff
@@ -1,5 +1,8 @@
 | Modifiable Custom Properties                             |
 | -------------------------------------------------------- |
+| `--mod-animation-duration-100`                           |
```

Comment:
My assumption is that we're seeing changes to the mods file for some of these because these may have been accidentally excluded before, is that right? Should these be included?

Reply:
These are similar to what Josh's update was seeing because we weren't running on compiled results.